newsModels/newsModels

```diff
@@ -0,0 +1,20 @@
+
+const db = require("../db/connection.js");
+//const { updateDataReversed } = require("../db/utility.js");
+
+exports.fetchTopics = (url) => {
+    return db.query("SELECT * FROM topics").then((topics) => {
+        console.log('models OK');
+        // console.log(url);
+        // console.log(topics);
+       
+        const formattedTopics = topics.rows.map(({ slug, description }) => {
```

Comment:
The data should not require any formatting, you are effectively destructuring, then restructuring the objects here

Reply:
I thought this way was given us at some lecture with a reason I cannot remember now. Should I change this code, or is it just for future use?